python/stack_plots.py

```diff
@@ -134,6 +141,12 @@
 samples.append("WW")
 samples.append("ZZ")
 samples.append("WZ")
+# PreUL NanoAODv7 samples for BFF comparison
+# Signal MC
+#samples.append("BFF")
+#samples.append("BFFdbs1p0")
+#samples.append("DY_v7")
+#samples.append("ttbar_v7")
```

Comment:
If we want to have the ability to plot the BFF samples, should we add the possibility to also plot those in BFF `mll` bins defined in `configuration_Zp.h`?

Reply:
You are right, I forgot to propagate that change. I will do so, thanks!